overrides/version.lua

```diff
@@ -1,3 +1,4 @@
 return {
-    expgaming = '6.0.0'
+    expgaming_lua = '6.0.0',
+    redmew_lua = '2019-02-24-76871ee'
```

Comment:
what does this do?

Reply:
This is related to sync code for our api. These values will be accessed by our api using /interface to decide if the roles need to be synced. So dont worry about it, its a core change rather than anything related to the minigames.